packages/gatsby/src/utils/eslint-config.js

```diff
@@ -7,6 +7,7 @@ module.exports = schema => {
       globals: {
         graphql: true,
         __PATH_PREFIX__: true,
+        __BASE_PATH__: true, // this will rarely, if ever, be used by consumers
```

Comment:
For more context here--it's used in the internal linking (e.g. with just pathPrefix) because pathPrefix itself is overloaded.